loleaflet/css/jsdialogs.css

```diff
@@ -753,14 +753,16 @@ input[type='checkbox']:checked.autofilter, .jsdialog input[type='checkbox']:chec
 	margin: -20px auto 0px;
 }
 
-#busypopup #busylabel {
+#busypopup #busylabel,
+.jsdialog  #busylabel {
 	margin: 20px;
 	text-align: center;
 	font-size: 100%;
 	text-transform: initial;
 }
 
-#busypopup #spinner-paper-g {
+#busypopup #spinner-paper-g,
+.jsdialog #spinner-paper-g {
```

Comment:
why not `#mobile-wizard #spinner-paper-g`? I fear jsdialog class will introduce some styling for desktop on mobile

Reply:
I need an additional class so I can effectively target `#mobile-wizard-content` on mobile. I will change the class thought to avoid any future problems.